.github/scripts/install-torch-tensorrt.sh

```diff
@@ -12,6 +12,8 @@ if [[ $(uname -m) == "aarch64" ]]; then
     install_cuda_aarch64
 fi
 
+pip install cuda-toolkit
```

Comment:
Is this not versioned? Can we find a place for this in the pyproject.toml or setup.py?